packages/desktop-client/src/components/payees/PayeeTable.tsx

```diff
@@ -28,56 +27,56 @@ type PayeeTableProps = {
   'onUpdate' | 'onViewRules' | 'onCreateRule'
 >;
 
-const PayeeTable = forwardRef<
-  ComponentRef<typeof Table<PayeeWithId>>,
-  PayeeTableProps
->(
-  (
-    { payees, ruleCounts, navigator, onUpdate, onViewRules, onCreateRule },
-    ref,
-  ) => {
-    const [hovered, setHovered] = useState(null);
-    const selectedItems = useSelectedItems();
+const PayeeTable = ({
+  tableRef,
```

Comment:
Curious why we should replace forwardRef here with a tableRef

Reply:
Makes the migration easier. Juggling `forwardRef` multiple layers deep with TS is very painful.